src/main.rs

```diff
@@ -786,6 +790,22 @@ fn try_main() -> Result<()> {
                 )?;
                 println!("Metadata file created at {path:?}");
             }
+            NonmemCommands::Sitrep { model_path } => {
+                if !model_path.exists() {
```

Comment:
sitrep shouldn't need a model path - this should be more against the environment itself relative to the lockfile to check on the general state of your system.